### ISSUE TITLE
Bug in S2T prep_mustc_data.py with --use-audio-input

### ISSUE THREAD

**Issue by johntsi** [facebookresearch/fairseq#3888]
## 🐛 Bug

In the following line:

https://github.com/pytorch/fairseq/blob/f6abcc2a67328bee8b15c596bb626ce2d720aae6/examples/speech_to_text/prep_mustc_data.py#L178

when the argument `use-audio-input` is True, the default values for the function `filter_manifest_df`

https://github.com/pytorch/fairseq/blob/f6abcc2a67328bee8b15c596bb626ce2d720aae6/examples/speech_to_text/data_utils.py#L244-L246

are filtering almost all the examples in MUST_C.

There should be different default values depending on the argument `use_audio_input`, since the default values `min_n_frames=5` and `max_n_frames=3000` reflect reasonable limits for spectrograms and not for waveforms.

**Comment by yuntang** [comment]
You should choose a bigger max_n_frames number. Assume your audio sampling rate 16k and the shift window for your feature frame is 10ms, then your corresponding max_n_frames should be changed to 3000*(10*16) = 480,000

**Comment by johntsi** [comment]
Yes, I agree. But the arguments `min_n_frames` and `max_n_frames` cannot be controlled when running `prep_mustc_data.py`. 
They should either be added to the argument parser or the function `filter_manifest_df` should additionally take as an argument the `args.use_audio_input` and use different defaults for min and max.

**Comment by yuntang** [comment]
@kahne can you take a look at this?
